setup.py

```diff
@@ -31,20 +31,20 @@
     packages=setuptools.find_packages(where="src"),
     python_requires=">=3.8",
     package_data={
-    	"pypromice.process": ["metadata.csv", "variables.csv"],
+    	"pypromice.ressources": ["variables.csv", "file_attributes.csv", "variable_aliases_GC-Net.csv"],
     	"pypromice.tx": ["payload_formats.csv", "payload_types.csv"],
         "pypromice.qc.percentiles": ["thresholds.csv"],
         "pypromice.postprocess": ["station_configurations.toml", "positions_seed.csv"],
     },
-    install_requires=['numpy>=1.23.0', 'pandas>=1.5.0', 'xarray>=2022.6.0', 'toml', 'scipy>=1.9.0', 'Bottleneck', 'netcdf4', 'pyDataverse==0.3.1', 'eccodes', 'scikit-learn>=1.1.0'],
+    install_requires=['numpy>=1.23.0,<2.0.0', 'pandas>=1.5.0', 'xarray>=2022.6.0', 'toml', 'scipy>=1.9.0', 'Bottleneck', 'netcdf4', 'pyDataverse==0.3.1', 'eccodes', 'scikit-learn>=1.1.0'],
```

Comment:
You can use tilde to define when you allow minor version updates.
`~=1.42` is equal to `>=1.42,<2` and `~=1.31.2` is equal to `>=1.31,<1.32`


```suggestion
    install_requires=['numpy~=1.23', 'pandas>=1.5.0', 'xarray>=2022.6.0', 'toml', 'scipy>=1.9.0', 'Bottleneck', 'netcdf4', 'pyDataverse==0.3.1', 'eccodes', 'scikit-learn>=1.1.0'],
```